### PR TITLE
refactor(platform-server): replace patch by CLI option.

### PR DESCRIPTION
**Pull request by JeanMeche**
The CLI introduced a new option `experimentalPlatform` to disable loading a polyfill from node. We don't need to patch it anymore.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,14 +48,13 @@
   },
   "// 1": "dependencies are used locally and by bazel",
   "dependencies": {
-    "@angular-devkit/build-angular": "19.0.0-next.6",
-    "@angular-devkit/core": "19.0.0-next.6",
-    "@angular-devkit/schematics": "19.0.0-next.6",
-    "@angular/build": "19.0.0-next.6",
+    "@angular-devkit/build-angular": "19.0.0-next.11",
+    "@angular-devkit/core": "19.0.0-next.11",
+    "@angular-devkit/schematics": "19.0.0-next.11",    "@angular/build": "19.0.0-next.11",
```
Comment (devversion) [inline package.json:53]: Nit: formatting is off here

**Review by devversion — APPROVED** [review]
LGTM (lint is failing)

Reviewed-for: global-dev-infra-approvers

**Review by alan-agius4 — APPROVED** [review]

**Comment by devversion** [comment]
@JeanMeche this seems to fail lint. Can you please look at it? thanks!

**Comment by josephperrott** [comment]
Caretaker Note: We can ignore the lint failure for the license check as this is a parsing issue for the license on a specific package.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 2e3e9b137af2090411ff80a05d26f3107f72d377.

The changes were merged into the following branches: main

**Pull request closed without merging**